scripts/goland-latest-indexing.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+# skip indexing in regular workspaces
+if [ ! "$GITPOD_HEADLESS" = "true" ] ; then exit ; fi
+
+# resolve latest JetBrains backend version running by Gitpod
+PRODUCT_CODE=GO
+curl -sSL "https://data.services.jetbrains.com/products?code=$PRODUCT_CODE&fields=distributions%2Clink%2Cname%2Creleases&_=$(date +%s)000" > /tmp/jb_products.xml
+JB_BACKEND_DOWNLOAD_URL=$(jq -r '.[0].releases[0].downloads.linux.link' /tmp/jb_products.xml)
+echo "$JB_BACKEND_DOWNLOAD_URL"
```

Comment:
We could possibly use `https://download.jetbrains.com/product?distribution=linux&code=$PRODUCT_CODE` here too. [[1](https://github.com/gitpod-io/gitpod/pull/9268)]

Reply:
Actually I'm not sure. It seems to have another semantic.